mmdet/models/necks/yolo_neck.py

```diff
@@ -109,7 +109,8 @@ def __init__(self,
         self.detect1 = DetectionBlock(in_channels[0], out_channels[0], **cfg)
         for i in range(1, self.num_scales):
             in_c, out_c = self.in_channels[i], self.out_channels[i]
-            self.add_module(f'conv{i}', ConvModule(in_c, out_c, 1, **cfg))
+            inter_c = out_channels[i - 1]
+            self.add_module(f'conv{i}', ConvModule(inter_c, out_c, 1, **cfg))
```

Comment:
Thanks for your contribution! I found the lack of unit testing, can you add it?

Reply:
Should I cover the whole YOLOV3Neck in the same manner as the other necks are covered? I'll try to do it in the next few days.

Reply:
It would be great if you could do that!

Reply:
I have added the unit test.